app.js

```diff
@@ -2,7 +2,7 @@
 // show extension is loaded
 console.log("[Better Mobile Youtube] Better Mobile Youtube has been Loaded!");
 
-
+let currentURL = window.location.href.replace(/\#.*/, '');
```

Comment:
can you please explain why did you add this, thanks.

Reply:
This regex gets rid of any # and everything after it, this means that URL changes are only detected when the actual webpage changes, not when a menu or popup is opened.